sourcemash/frontend/static/js/views/item_card_view.js

```diff
@@ -70,16 +93,20 @@ Sourcemash.Views.ItemCardView = Backbone.View.extend({
             this.model.save({saved: false},
                 {success: this.savedToast});
         } else {
-            this.model.save({saved: true},
-                {success: this.savedToast});
-
-            mixpanel.track("Saved", { "Item Title": this.model.get('title'),
-                                    "Feed Title": this.model.feed.get('title') })
-
-            if (!this.model.feed.get('subscribed')) {
-                this.showSubscribeModal({'source':'saved'});
-            }
-        }
+            if (!this.user.get('id')) {
```

Comment:
You can't put the modal in the side nav b/c it won't load in front of the whole page if you do. It will only load in the sidenav, which means it get's cut off when you enter the "body" section of the page.